cmd/dvotecli/commands/entity.go

```diff
@@ -0,0 +1,63 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"go.vocdoni.io/dvote/client"
+	"go.vocdoni.io/dvote/router"
+	"go.vocdoni.io/dvote/types"
+	"nhooyr.io/websocket"
+)
+
+var (
+	entityCmd = &cobra.Command{
+		Use:   "entity list",
+		Short: "entity subcommands",
+	}
+	entityListCmd = &cobra.Command{
+		Use:   "list [prefix]",
+		Short: "list entities matching an optional prefix",
+		RunE:  entityList,
+	}
+)
+
+func init() {
+	rootCmd.AddCommand(entityCmd)
+	entityCmd.AddCommand(entityListCmd)
+}
+
+func entityList(cmd *cobra.Command, args []string) error {
+
+	cl, err := client.New(host)
+	if err != nil {
+		return err
+	}
+	defer cl.Conn.Close(websocket.StatusNormalClosure, "")
+
+	var req types.MetaRequest
+	req.Method = "getEntities"
+	resp, err := cl.Request(req, nil)
+	if err != nil {
+		return err
+	}
+	if !resp.Ok {
+		return fmt.Errorf(resp.Message)
+	}
+	entities := resp.EntityIDs
```

Comment:
Here you are just copying the pointer, this is not a deep copy. So I'm not sure what would be the behavior when the loop changes the content of resp.EntityIDs. I think the correct way to do this is:
```
entities  := []string{}
entities = append(entities, resp.EntityIDs...)
if len(entities) == .....
```